### ISSUE TITLE
No Site menu on sub-networks multisites with WP Multi-network

### ISSUE THREAD

**Issue by danaskallman** [meitar/multisite-directory#15]
When using with [WP multi-network plugin](https://github.com/stuttter/wp-multi-network), documented setup in [more detail here](https://plan.glocal.coop/projects/activitstnetwork/wiki/Multisite#Configure-WordPress-Multi-Network). What the plugin does it add a Network menu and allow on a click to create a new multisite on the same codebase and DB, each has it's own network admin and main site with sub-site like a regular multisite. The plugins are enabled on a network by network level. 

When I network enable the Multisite Directory plugin on the main network it works as intended, Categories menu in Site of the Network and the main site has the Sites admin menu. But once I do this on any sub-network, I only the the network admin Sites menu but not the Sites menu on the main site admin of that second/third/etc network multisite. Thoughts on why this may be?

**Comment by fabacab** [comment]
Ohhhhh yeah, you're setting up a Multi-NETWORK.

I'll have to add this config to the development environment setup before I can look at the bug in this specific plugin. Thanks for the heads up.

**Comment by fabacab** [comment]
Just a note to say I got the `vvv-activistnetwork` repo working with WP Multi-Network capabilities as of commit [5a3e7036ad13c3a4a54e6b40459e71a469fefd16](https://github.com/meitar/vvv-activistnetwork/commit/5a3e7036ad13c3a4a54e6b40459e71a469fefd16), so will have a look at this bug shortly.

**Comment by danaskallman** [comment]
Awesome, I wasn't able to get setup locally before so excited to try this out?

Question in terms of aggregating data to the parent network. Is there a way to allow for the parent network to show/display/access each network directory to show who is involved or who is member? It would vary by multi-network, but would be awesome to be able to have that ability. Also not sure if it applies here or is it's own thing.

**Comment by fabacab** [comment]
After looking through the code for the WP Multi-Network plugin I believe it's possible in a similar way as core's `switch_to_blog()` function works. We should track that in a new issue. Can you elaborate on what you're hoping to make possible in a new feature request ticket?

**Comment by danaskallman** [comment]
Wanted to see if there was any thoughts/feedback on what we would need to do to use on a multinetwork. I am attempting to setup and am stuck on this issue.

**Comment by fabacab** [comment]
Sorry, Dana, I've neglected this for a little while. At first I was waiting for `vv` to accept my PR bradp/vv#267 but then I got distracted with other things. I haven't looked at this bug but if you're blocked by it today I will prioritize taking a look ASAP.

**Comment by fabacab** [comment]
@danaskallman Can you give [this patched version](https://github.com/meitar/multisite-directory/archive/00f340294e1dc306a9347273c566687d3c0614ea.zip) of the plugin a go and see if it resolves the issues in your Multi-Network environment? Thanks and sorry again for the delay on this bug. :)

**Comment by danaskallman** [comment]
No worries much appreaciated. It does seem to resolve that issue, I can no network activate on each instance of the multinetwork and they can manage their own sites directory. 

Thanks!

**Comment by fabacab** [comment]
Just to clarify, you mean to say you _can_ now use the plugin in a multi-network, yes? (I think you typo'd there somewhere.)

**Comment by danaskallman** [comment]
Yeah, it was supposed to say "I can now" activate on each multisite instance of multi-network.

**Comment by fabacab** [comment]
Cool, I'll tag and release sometime this week. :)
